### PR TITLE
allow express to bind to specific address.

### PR DESCRIPTION
**Pull request by OSPFNeighbour**
yea... that.

### CODE REVIEW

File: server/src/main.js [hunk 1]
```diff
@@ -36,6 +37,6 @@ app.use('/graphiql', graphiqlExpress({
 
 const graphQLServer = createServer(app);
 
-graphQLServer.listen(port, () => {
+graphQLServer.listen(port,GRAPHQL_ADDRESS, () => {
```
Comment (sdunster) [inline server/src/main.js]: plz 2 whitespace
```
graphQLServer.listen(port, GRAPHQL_ADDRESS, () => {
```

**Comment by OSPFNeighbour** [comment]
Windows seems to bind differently, change was made due to someone reporting
issues with default bind.

0.0.0.0 was a holder as I listed a specific address I wanted to bind to.

**Comment by sdunster** [comment]
Please figure out the details so we can make sure this is actually the right thing to do. It might be better just to special case this for windows if absolutely necessary.

**Comment by sdunster** [comment]
Luke Quinane - Yesterday at 1:47 PM
> re the windows binding issue. I've double checked that and it was me just getting mixed up between the graphql server which was bound to 8080 and something else which was bound to 8081

I don't think we need this change so I'm going to close it but please feel free to reopen if you think it's important.

**Pull request closed without merging**